VideoAPI/Video.API/Controllers/AudioRecordingController.cs

```diff
@@ -224,7 +224,7 @@ public async Task<IActionResult> GetAudioRecordingLinkAsync(Guid hearingId)
                 return NotFound();
             }
 
-            var audioFileLink = await _storageService.CreateSharedAccessSignature(filePath, TimeSpan.FromDays(7));
+            var audioFileLink = await _storageService.CreateSharedAccessSignature(filePath, TimeSpan.FromDays(14));
```

Comment:
Is it possible to test the signature contains the 14 day timespan?

Reply:
not unit test, but yes acceptance test, which are being done by Nick now